roq-simulation/src/main/java/org/roq/simulation/test/RoQDockerTestCase.java

```diff
@@ -0,0 +1,94 @@
+/**
+ * Copyright 2012 EURANOVA
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * 
+ */
+package org.roq.simulation.test;
+
+import java.net.ConnectException;
+
+import org.apache.log4j.Logger;
+import org.junit.After;
+import org.junit.Before;
+import org.roq.simulation.RoQDockerLauncher;
+import org.roqmessaging.client.IRoQQueueManagement;
+import org.roqmessaging.core.factory.RoQConnectionFactory;
+import org.roqmessaging.core.factory.RoQQueueManager;
+
+/**
+ * Class RQDockerTestCase
+
+ * @author bvanmelle
+ */
+public class RoQDockerTestCase {
+	protected RoQDockerLauncher launcher = null;
+	protected Logger logger = Logger.getLogger(RoQDockerTestCase.class);
+	protected static RoQConnectionFactory connection = null;
+	protected static IRoQQueueManagement queueManager = null;
+	
+	 /**
+	 * @throws java.lang.Exception
+	 */
+	@Before
+	public void setUp() throws Exception {
+		this.logger.info("Setup TEST with Docker containers");
+		Thread.sleep(3000);
+		this.launcher = new RoQDockerLauncher();
+		this.launcher.setUp();
+		Thread.sleep(1000);
+		String zkConnectionString = launcher.getZkConnectionString();
+		queueManager = new RoQQueueManager(zkConnectionString, 9, 5000);
+		connection = new RoQConnectionFactory(zkConnectionString, 9, 5000);
```

Comment:
Is there a reason for not using the default nbRetries and timeout for the RoQConnectionFactory here? It's the only place where you use the constructor with 3 arguments, so if it's not needed you can remove that constructor (less code is better).